### PR TITLE
Bug 1203898 - [browser] Re-skin Firefox Account sign in process for T…

### PR DESCRIPTION
**Pull request by danhuang1202**
…V, r=yifan
- re-skin fxa-email and fxa-enter-password page

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d5643d03bbb6e8100afa1d3c0c840d31064ddbd4

File: tv_apps/smart-system/fxa/js/screens/fxam_enter_email.js [hunk 1]
```diff
@@ -88,19 +89,35 @@ var FxaModuleEnterEmail = (function() {
...
     this.fxaEmailInput.addEventListener(
       'blur',
       function onBlur() {
         this.fxaLogo.removeAttribute('hidden');
+        this.fxaEmailCleanBtn.classList.remove('show');
+      }.bind(this)
+    );
+
+    this.fxaEmailCleanBtn.addEventListener(
+      'mousedown',
```
Comment (begeeben) [inline tv_apps/smart-system/fxa/js/screens/fxam_enter_email.js]: We need to support key events as well. Please change it to ```click```, and check whether it's mouse left button or enter key.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9efb2c32e6bab643f4a106dcd5c415cd8ac069ab

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f90ba707414828062e8d0013662c5ae386d10e44

**Pull request merged**